libs/langchain-community/src/llms/hf.ts

```diff
@@ -107,40 +117,72 @@ export class HuggingFaceInference extends LLM implements HFInput {
     return "hf";
   }
 
+  invocationParams(_options?: this["ParsedCallOptions"]) {
+    return {
+      model: this.model,
+      parameters: {
+        // make it behave similar to openai, returning only the generated text
+        return_full_text: false,
+        temperature: this.temperature,
+        max_new_tokens: this.maxTokens,
+        stop_sequences: this.stopSequences,
```

Comment:
Since we're passing `options` through, we should prioritize the passed option, and fallback to the class property.
This should apply to the rest of the `parameters` too.
```typescript
options.stop ?? this.stop
```